apps/etterlatte-behandling/src/main/kotlin/sak/SakService.kt

```diff
@@ -185,6 +187,8 @@ class SakServiceImpl(
         return dao.hentSak(id).sjekkEnhet()
     }
 
+    override fun finnFlyktningForSak(id: Long): Flyktning? = dao.finnFlyktningForSak(id)
```

Comment:
Gjør det noe at denne vil svare for saker hvor sbh. ikke har gyldig enhet?

Reply:
Godt poeng. Bør nok sjekke om saksbehandler har tilgang til enheten saken ligger på.